app/javascript/components/Dashboard/Tooltips.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Tooltip = ({ feature }) => {
```

Comment:
Is this Tooltip specific to the map? Can it be used by other react components? I mean is it generic?

Reply:
Custom component, we will keep it for now.

Reply:
Got it, looks good then